code/controllers/subsystem/shuttles.dm

```diff
@@ -254,7 +254,7 @@ var/datum/subsystem/shuttle/SSshuttle
 	slip.ordernumber = ordernum
 
 	var/stationName = (errors & MANIFEST_ERROR_NAME) ? new_station_name() : station_name()
-	var/packagesAmt = SSshuttle.shoppinglist.len + ((errors & MANIFEST_ERROR_COUNT) ? 0 : rand(1,2))
```

Comment:
oh my fucking god ignore me
I have had like a 20hour day.
You are correct
Wow I feel dumb.